src/lib/y2network/config_reader/sysconfig.rb

```diff
@@ -64,13 +64,19 @@ def find_interfaces
       # return [RoutingTable] an object with routes
       def load_routes
         # load /etc/sysconfig/network/routes
-        routes = SysconfigRoutesReader.new.config
+        routing_table = SysconfigRoutesReader.new.config
         # load /etc/sysconfig/network/ifroute-*
-        dev_routes = find_interfaces.map do |iface|
-          SysconfigRoutesReader.new(routes_file: "/etc/sysconfig/network/ifroute-#{iface.name}").config
+        dev_routing_tables = find_interfaces.map do |iface|
+          SysconfigRoutesReader.new(
+            routes_file: "/etc/sysconfig/network/ifroute-#{iface.name}"
+          ).config
         end
 
-        dev_routes.inject(routes) { |a, e| a.concat(e.routes) }.uniq
+        dev_routing_tables.reduce(routing_table) do |rt, dev_rt|
+          rt.concat(dev_rt.routes)
+        end
+        routing_table.routes.uniq!
+        routing_table
```

Comment:
this is not needed, isn't it?

Reply:
yes, if you want to return a `RoutingTable`. Another approach might to filter the routes when calling to `#concat` to be sure that they are not repeated.

Reply:
ok